<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
 </author>
 <class>VideoImportDialog</class>
 <widget class="QWidget" name="VideoImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>493</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Video Animation Import</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,6,4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_1">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="fileLocationLabel">
             <property name="text">
              <string>Choose a video file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="filePickerButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="thumbnailImageHolder">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>200</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>200</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="text">
                <string>No Preview</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <widget class="QSlider" name="videoPreviewSlider">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximum">
                  <number>1</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="videoPreviewSliderValueLabel">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="fileLoadedDetails">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="text">
                <string>File details...</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="margin">
                <number>3</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QSpinBox" name="currentFrameNumberInput">
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="maximum">
                <number>99999</number>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QPushButton" name="prevFrameButton">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="nextFrameButton">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>49</width>
                   <height>49</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QGroupBox" name="exportOptionsGroup">
             <property name="title">
              <string>Export Options</string>
             </property>
             <widget class="QWidget" name="layoutWidget_3">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>27</y>
                <width>411</width>
                <height>101</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <property name="sizeConstraint">
                <enum>QLayout::SetMinimumSize</enum>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="fpsLabel">
                 <property name="text">
                  <string>FPS:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="fpsSpinbox"/>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="fpsDocumentLabel">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QLabel" name="startExportingAtLabel">
                 <property name="text">
                  <string>Start at: </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QDoubleSpinBox" name="startExportingAtSpinbox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="frameSkipLabel">
                 <property name="text">
                  <string>Skip Interval:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="frameSkipSpinbox"/>
               </item>
               <item row="1" column="3">
                <widget class="QLabel" name="exportDurationLabel">
                 <property name="text">
                  <string>Duration:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="4">
                <widget class="QDoubleSpinBox" name="exportDurationSpinbox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="importDocumentLabel">
                 <property name="text">
                  <string>Import into:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="1" colspan="2">
                <widget class="QComboBox" name="cmbDocumentHandler"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOptions">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5" stretch="3,2,2">
         <item>
          <widget class="QGroupBox" name="optionsDocumentGroup">
           <property name="title">
            <string>Document</string>
           </property>
           <widget class="QWidget" name="layoutWidget_5">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>441</width>
              <height>101</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,0,0,0">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <item row="0" column="3">
              <widget class="QComboBox" name="cmbDocumentColorModel"/>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="documentHeightSpinbox"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="documentColorDepthLabel">
               <property name="text">
                <string>Depth:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="documentWidthSpinbox"/>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="documentColorModelLabel">
               <property name="text">
                <string>Model:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="documentWidthLabel">
               <property name="text">
                <string>Width:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="documentHeightLabel">
               <property name="text">
                <string>Height:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="documentResolutionLabel">
               <property name="text">
                <string>Resolution:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QComboBox" name="cmbDocumentColorDepth"/>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="documentColorProfileLabel">
               <property name="text">
                <string>Profile:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QComboBox" name="cmbDocumentColorProfile"/>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="documentResolutionSpinbox"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="optionsImportVideoScaleGroup">
           <property name="title">
            <string>Import Video Scale</string>
           </property>
           <widget class="QWidget" name="layoutWidget_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>441</width>
              <height>71</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0,0">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <item row="0" column="2">
              <widget class="QLabel" name="videoScaleFilterLabel">
               <property name="text">
                <string>Filter:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="videoWidthLabel">
               <property name="text">
                <string>Width:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="videoHeightLabel">
               <property name="text">
                <string>Height:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="videoWidthSpinbox"/>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="videoHeightSpinbox"/>
             </item>
             <item row="0" column="3">
              <widget class="QComboBox" name="cmbVideoScaleFilter"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="optionsFFMpegGroup">
           <property name="title">
            <string>FFMpeg</string>
           </property>
           <widget class="QWidget" name="gridLayoutWidget_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>441</width>
              <height>71</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0,0,0,0" columnminimumwidth="0,0,0,0,0">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="ffmpegLocationLabel">
               <property name="text">
                <string>FFMPeg Path:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="cmbFFMpegLocation"/>
             </item>
             <item row="0" column="4">
              <widget class="QToolButton" name="ffmpegPickerButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="ffprobeLocationLabel">
               <property name="text">
                <string>FFProbe Path:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="cmbFFProbeLocation"/>
             </item>
             <item row="1" column="4">
              <widget class="QToolButton" name="ffprobePickerButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
